web_menu_collapsible/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Collapsible menu
+================
+
+This module makes all menus collapsible for all users.
+
+
+Configuration
+=============
+
+No configuration is needed.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/8.0
+
+Bug Tracker
```

Comment:
Add a section "Known issues / Roadmap" indicating: "There's no visual hint about the collapsed/uncollapsed state of a menu on second level menus"